src/components/RecipeItems.js

```diff
@@ -0,0 +1,45 @@
+
+// import axios from "axios";
+// import { useEffect, useState } from "react";
+// import { useParams } from 'react-router-dom';
+
+
+
+// const RecipeItems = () => {
```

Comment:
I wish I could actually help with error handling feedback, but I know so little! It does looks like there's a CORS error happening when the user tries to submit more than one request - I feel like you might need a proxy server to solve this? I'm sure there is something in the notes about this somewhere/I'm sure a mentor could help you set one up!